.buildkite/scripts/sudo-integration-tests.sh

```diff
@@ -15,7 +15,9 @@ if [[ -n "$PACKAGE_VERSION" ]]; then
     PACKAGE_VERSION=${PACKAGE_VERSION}"-SNAPSHOT"
 fi
 set +e
-AGENT_VERSION="${PACKAGE_VERSION}" SNAPSHOT=true TEST_DEFINE_PREFIX="sudo_${GROUP_NAME}_ubuntu" gotestsum --no-color -f standard-quiet --junitfile "build/${GROUP_NAME}.integration.xml" --jsonfile "build/${GROUP_NAME}.integration.out.json" -- -tags integration -test.shuffle on -test.timeout 2h0m0s -test.run "${TESTS_TO_RUN}" github.com/elastic/elastic-agent/testing/integration
+-integration.groups="default"
+go test -tags integration github.com/elastic/elastic-agent/testing/integration -v -args -integration.groups="${GROUP_NAME}"
```

Comment:
Do you plan to supply ESS params here as discussed or is that something for a follow up PR?

Reply:
IIUC the question: the ESS params are defined here: https://github.com/elastic/elastic-agent/pull/5900/files#diff-5d335dc279ba837a8bf759aeed11e1a2cd97235b2355ee804e70531db5fdffe4R38. Buildkite passes all required ESS data from the previous step

Reply:
Right now ESS params can be passed via env vars, we can also support CLI arguments (with the small caveat of passwords being visible on a command line)

Reply:
Env vars is fine, thanks for clarifying.